### ISSUE TITLE
Task for publishing and closing staging repository

### ISSUE THREAD

**Issue by yooksi** [gradle-nexus/publish-plugin#77]
There should be a single Gradle task that both published and closes staging repository.

For now we have to run the following Gradle command to achieve this:
```
./gradlew publishToSonatype closeSonatypeStagingRepository
```
It would be a lot simpler to have an aggregate task that could be called like such:
```
./gradlew publishToSonatypeAndCloseStagingRepository
```

**Comment by szpak** [comment]
Looking at the implementation details publishing is somehow different operation than releasing. It can be done separately or with different configuration. Would you also want to have one for `publishToSonatypeAndCloseAndReleaseStagingRepository`?

**Comment by yooksi** [comment]
> Would you also want to have one for `publishToSonatypeAndCloseAndReleaseStagingRepository` ?

That would also be useful, perhaps it can be called `publishToSonatypeAndReleaseStagingRepository`.

**Comment by szpak** [comment]
> That would also be useful, perhaps it can be called `publishToSonatypeAndReleaseStagingRepository`.

This is somehow inconsistent with other tasks. In the context of preceding publishing the release operation has to be done before the release on, but in general `release` can be used on its own (e.g. if the new version is published and staging repository is closed and after manual testing that version is being released - in a separate Gradle call, even days after the previous operation).

In the past we had also requests to limit number of tasks, with the argument used that if needed, it is easy to create a small orchestrating tasks (with `dependsOn`), while removing or hiding task is somehow more complicated.

Personally, I'm not a big fan of adding that task(s), but let's wait for opinions of the other users.

**Comment by marcphilipp** [comment]
I don't think we need this combined task. The more task combinations we add the more confusing it gets, doesn't it?
